cmd/tls/handlers/utils.go

```diff
@@ -95,11 +95,7 @@ func nodeFromEnroll(req types.EnrollRequest, env environments.TLSEnvironment, ip
 		ConfigHash:      req.HostDetails.EnrollOsqueryInfo.ConfigHash,
 		BytesReceived:   recBytes,
 		RawEnrollment:   string(enrollRaw),
-		LastStatus:      time.Time{},
-		LastResult:      time.Time{},
-		LastConfig:      time.Time{},
-		LastQueryRead:   time.Time{},
-		LastQueryWrite:  time.Time{},
```

Comment:
The node.html Template also needs updating now I guess?

Reply:
Good catch, I completely missed that. Let me prepare a quick PR to make sure `osctrl-admin` does not break. Thanks!